LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016   
+Crown Copyright (c) 2016-2017
```

Comment:
This a good view from US side - http://www.photoattorney.com/qa-what-year-should-i-use-for-my-copyright-notice/